selfdescribe.json

```diff
@@ -55033,6 +55033,9 @@
         "guest_id": {
           "description": "For virtual machine metrics, the guest identifier of the virtual machine to which the metric pertains."
         },
+        "host": {
+          "description": "For virtual machine metrics, the host dimension will be set to the VM IP instead of the running agent hostname. This behavior is configurable, check the `vmHostDimension` option for details."
```

Comment:
```suggestion
          "description": "For virtual machine metrics, the host dimension will be set to the VM IP instead of the running agent hostname. This behavior is configurable; check the `vmHostDimension` option for details."
```